rn/Teacher/ios/Teacher/SpeedGrader/SubmissionGrader.swift

```diff
@@ -183,73 +185,73 @@ struct SubmissionGrader: View {
 
     enum GraderTab: Int, CaseIterable { case grades, comments, files }
 
-    func tools(bottomInset: CGFloat) -> some View {
-        VStack(spacing: 0) {
-            Picker(selection: Binding(get: { tab }, set: { newValue in
-                if drawerState == .min {
-                    snapDrawerTo(.mid)
-                }
-                withAnimation(.default) {
-                    tab = newValue
-                }
-            }), label: Text(verbatim: "")) {
-                Text("Grades").tag(GraderTab.grades)
-                Text("Comments").tag(GraderTab.comments)
-                if selected.type == .online_upload, let count = selected.attachments?.count, count > 0 {
-                    Text("Files (\(count))").tag(GraderTab.files)
-                } else {
-                    Text("Files").tag(GraderTab.files)
-                }
+    @ViewBuilder
+    func tools(bottomInset: CGFloat, isDrawer: Bool) -> some View {
+        Picker(selection: Binding(get: { isDrawer && drawerState == .min ? nil : tab }, set: { newValue in
```

Comment:
This is a big diff but seems the main difference is that the Picker is no longer embedded in a `VStack` (assuming to fix GeometryReader bugs)

Reply:
yep, that helps iOS 13 a lot.